website/src/_posts/2020-03-01-beam-2.20.0.md

```diff
@@ -0,0 +1,76 @@
+---
+layout: post
+title:  "Apache Beam 2.20.0"
+date:   2020-03-01 00:00:01 -0800
```

Comment:
Feel like we should also update this date.

Reply:
Done